SharpBucket/V2/Pocos/Links.cs

```diff
@@ -10,6 +10,7 @@ public class Links
         public Link followers { get; set; }
         public Link avatar { get; set; }
         public Link following { get; set; }
+        public Link html { get; set; }
```

Comment:
The Links POCO is too wide and we try to not use it in new POCOs and prefer to create dedicated classes.
So can you create an IssueLinks class and put in it just the link properties that are returned with issues ?

Reply:
I could create `IssueLinks` and inherit from `Links` and just add `html`, but I think most of their API objects include an `html` link?

Reply:
No, Src resource do not return html links for example ;)

Reply:
Done.